apps/fallback-ensapi/package.json

```
@@ -0,0 +1,40 @@
+{
+  "private": true,
+  "name": "fallback-ensapi",
+  "version": "0.1.0",
+  "type": "module",
+  "description": "ENSNode Fallback API",
```

```suggestion
  "description": "Infrastructure-level ENSApi fallback",
```

Goal: This provides a fallback to ENSApi but not other services within an ENSNode instance.